notebooks/Image_Analysis.ipynb

```diff
@@ -109,6 +110,58 @@
     "\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "7b118a48",
+   "metadata": {},
+   "source": [
+    "#### Box Plot Code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19f35cb9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "box_plot_data=[bg_vars_a, bg_vars_b]\n",
+    "\n",
+    "fig, ax = plt.subplots(dpi=100, figsize=(18,10))\n",
+    "\n",
+    "box = ax.boxplot(box_plot_data,vert=0,patch_artist=True,labels=['PSP-A','PSP-B'], \n",
+    "          showmeans=True, meanline=True, showfliers=False,)\n",
+    " \n",
+    "colors = ['red', 'blue',]\n",
+    "for patch, color in zip(box['boxes'], colors):\n",
+    "    patch.set_facecolor(color)\n",
+    "    patch.set_alpha(0.55)\n",
+    "\n",
+    "for median in box['medians']:\n",
+    "    median.set_color('black')\n",
+    "for i, means in enumerate(box['means']):\n",
+    "    means.set_color('black')\n",
+    "    x, y = means.get_xydata()[1]\n",
+    "    mean = np.array(box_plot_data[i]).mean()\n",
+    "    text = ' μ={:.2f}'.format(mean)\n",
+    "    ax.annotate(text, xy=(x, y+0.05), fontsize=15)\n",
+    "    \n",
+    "\n",
+    "plt.plot([], [], '--', linewidth=1, color='black', label='Mean')\n",
+    "plt.plot([], [], '-', linewidth=1, color='black', label='Median')\n",
+    "\n",
+    "plt.legend(fontsize=15)\n",
+    "\n",
+    "\n",
+    "plt.xticks(fontsize=15 )\n",
+    "plt.yticks(fontsize=15 )\n",
+    "plt.xscale('log')\n",
+    "plt.minorticks_off()\n",
+    "plt.xlabel(\"Var of Laplacian of background in log scale\", fontsize=18)\n",
+    "plt.ylabel(\"Dataset\", fontsize=18)\n",
+    "plt.show()"
+   ]
+  },
```

Comment:
For Spectral Analysis, add support for plotting more than one dataset.

Reply:
```
fig, ax = plt.subplots(dpi=300)
ax.plot(avg_psd_1d_a, color="red", label="PSD-A")
ax.plot(avg_psd_1d_b, color="blue", label="PSD-B")
ax.set_xscale('log')
ax.set_yscale('log')
plt.xlim([1, None])
```
Added this.